tests/unit/test_crl.cpp

```diff
@@ -176,3 +208,23 @@ TEST_F(CRLTest, testConvertionFromAndToPEMIsNoOp)
 {
     EXPECT_EQ(CertificateRevocationList::fromPEM(rootCrl->toPEM()).toPEM(), rootCrl->toPEM());
 }
+
+TEST_F(CRLTest, testLoadCrlPemChain)
+{
+    auto crlList = mococrw::util::loadCrlPEMChain(crlPemChain);
+
+    //Issuer: CN = CRL Sub CA 1, C = DE, OU = BMW Car IT, O = BMW
+    auto issuerName =crlList.at(0).getIssuerName();
+    EXPECT_EQ("CRL Sub CA 1", issuerName.commonName());
+    EXPECT_EQ("DE", issuerName.countryName());
+    EXPECT_EQ("BMW", issuerName.organizationName());
+    EXPECT_EQ("BMW Car IT", issuerName.organizationalUnitName());
+
+    //Issuer: C = PT, ST = Porto, L = Porto, O = CTW, OU = JC7
```

Comment:
Please avoid BMW specific names in new tests.

Reply:
Agreed, Removed.